.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
```

Comment:
Should it be `'bug'` or `bug`? In the feature request template you used `'`.

Reply:
this one got auto created, so i guess it should be bug, rather than 'bug' -- i'll change the other one